notebooks/contributing/formatting.md

```diff
@@ -0,0 +1,289 @@
+[simpeg.potential_fields.gravity](xref:simpeg#simpeg.potential_fields.gravity). We also
+import some useful utility functions from [simpeg.utils](xref:simpeg#simpeg.utils).
+To simulate gravity data, we need to define our problem geometry on a numerical grid
+(or mesh). To generate the mesh, we used the
+[discretize](https://discretize.simpeg.xyz/en/main) package.
+:::
+
+### Cell 2: Code Cell
+
+The coding cell where functionality is imported should be organized. E.g.
+
+```python
+# SimPEG functionality
+from simpeg.potential_fields import gravity
+from simpeg.utils import plot2Ddata, model_builder
+from simpeg import maps
+
+# discretize functionality
+from discretize import TensorMesh
+from discretize.utils import mkvc, active_from_xyz
+
+# Common Python functionality
+import numpy as np
+from scipy.interpolate import LinearNDInterpolator
+import matplotlib as mpl
+
+mpl.rcParams.update({"font.size": 14})
+import matplotlib.pyplot as plt
+import os
+```
+
+## Tutorial Sections (and Subsections)
+
+Each section (or subsection) is constructed using a combination of Markdown and code cells. The structure and formatting of these are discussed below.
+
+### Header and Summary
+
+Each section (or subsection) begins with a Markdown cell containing the header and a summary.
+Here, the contributor must:
+
+* summarize what is being done in the section
+* describe any new functionality that is being introduced
+* provide links to all relevant API documentation
+* describe the choices for hyperparameter values used in the tutorial
+
+E.g.
+
+```
+## Define the Survey
+
+Surveys within SimPEG generally require the user to create and connect three
+types of objects:
+
+- [receivers](xref:simpeg#simpeg.potential_fields.gravity.receivers.Point):
+which define the locations of field measurements and type of data being measured.
+- [sources](xref:simpeg#simpeg.potential_fields.gravity.sources.SourceField):
+the passive or active sources responsible for generating geophysical responses,
+and their associated receivers.
+- [survey](xref:simpeg#simpeg.potential_fields.gravity.survey.Survey): the object
+which stores and organizes all of the sources and receivers.
+
+Here, we define the survey that will be used for the forward simulation. Gravity
+surveys are simple to create. The user only needs an (N, 3)
+[numpy.ndarray](xref:numpy#numpy.ndarray) to define the xyz locations of the
+observation locations, and a list of field components
+which are to be measured. For the tutorial simulation, the receivers are located
+5 m above the surface topography and spaced 10 m apart.
+```
+
+If something has been thoroughly explained in an introductory tutorial, you can avoid repetition by linking to that tutorial.
+However, the choice in values used in your tutorial should always be stated. E.g.
+
+```
+## Assign Uncertainties
+
+Approaches for applying reasonable uncertainties to normalized voltage and apparent
```

Comment:
```suggestion
```markdown
## Assign Uncertainties

Approaches for applying reasonable uncertainties to normalized voltage and apparent
```